docs/v2-release-notes.md

```diff
@@ -2,6 +2,21 @@
 
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
+## Jul 8, 2024
+
+A new [JDBC driver](https://github.com/timeplus-io/timeplus-native-jdbc) for Timeplus is now available, supporting both streaming and batch queries. Compared to the JDBC driver we released last year, this new driver talks to Timeplus via the TCP native  protocol, with better performance. [Learn more](jdbc)
```

Comment:
`native  protocol` are there two spaces?

Reply:
Fixed - thank you Calvin